README.md

```diff
@@ -475,48 +477,67 @@ curl -v \
+Example request for sending attachment-based conversation History Request, be sure to replace placeholder values `<..>`
 
 ```bash
-curl --location 'http://localhost:8085/api/v1/attachmentHistory' \
---header 'OrgId: <orgId>>' \
---header 'AuthorizationContext: <AuthorizationContext>' \
---header 'RequestId: <RequestId>' \
---header 'AuthorizationContextType: <AuthorizationContextType>' \
---header 'Authorization: Bearer <AccessToken>' \
---form 'sendHistoryAttachmentsRequestJson="{
-        \"channelAddressIdentifier\": \"<channelAddressIdentifier>\",
-        \"conversationParticipants\": [
-            {
-                \"displayName\": \"<displayName>\",
-                \"participant\": {
-                    \"subject\": \"<participantSubject>\",
-                    \"role\": \"<ParticipantRole>\",
-                    \"appType\": \"<AppType>\"
-                },
-                \"joinedTime\": <Timestamp in Unix Epoch Milliseconds>
-            }
-        ],
-        \"conversationEntries\": [
-            {
-                \"clientTimestamp\": <Timestamp in Unix Epoch Milliseconds>,
-                \"sender\": {
-                    \"subject\": \"<senderSubject>\",
-                    \"role\": \"<ParticipantRole>\",
-                    \"appType\": \"<AppType>\"
-                },
-                \"entryPayload\": {
-                    \"id\": \"<messageId>\",
-                    \"attachmentIndex\": <attachmentIndex>,
-                    \"contentLength\": <contentLength>,
-                    \"text\": \"Hi There\"
+curl -v \
+-H "Authorization: Bearer <AccessToken>" \
+-H "content-type: application/json" \
+-H "OrgId: <OrgId>" \
+-H "AuthorizationContext: <AuthorizationContext>" \
+-H "AuthorizationContextType: <AuthorizationContextType>" \
+-H "RequestId: <RequestId>" \
+-X POST -d "{
+  \"channelAddressIdentifier\": \"<channelAddressIdentifier>\",
+  \"conversationParticipants\": [
+    {
+      \"displayName\": \"<displayName>\",
+      \"participant\": {
+            \"subject\": \"<unique identifier for the participant>\",
+            \"role\": \"<ParticipantRole>\",
+            \"appType\": \"<AppType>\"
+        },
+      \"joinedTime\":  <Timestamp in Unix Epoch Milliseconds>
+    }
+  ],
+  \"conversationEntries\": [
+    {
+        \"clientTimestamp\":  <Timestamp in Unix Epoch Milliseconds>,
+        \"sender\": {
+            \"subject\": \"<unique identifier for the participant>\",
+            \"role\": \"<ParticipantRole>\",
+            \"appType\": \"<AppType>\"
+        },
+        \"entryPayload\": {
+            \"entryType\": \"Message\",
+            \"id\": \"<payloadId>\",
+            \"abstractMessage\": {
+                \"messageType\": \"StaticContentMessage\",
+                \"inReplyToMessageId\": <inReplyToMessageId>,
+                \"id\": \"<messageId>\",
+                 \"references\": [{
+                    \"recordId\": \"068LT000000DoS1\",
```

Comment:
we need to let customers know which records' ID is this. Please find the entity name and mention here

Reply:
updated